docs/HyperIndex/migration-guide.md

```
@@ -163,9 +163,11 @@ subscription.position = event.params.tokenId
 subscription.save()
 }
 
+
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)